src/Surfnet/ServiceProviderDashboard/Application/Service/ServiceConnectionService.php

```diff
@@ -217,4 +227,39 @@ private function getTeam(MetaData $metadata): string
         }
         return $team;
     }
+
+    private function getContactData(ManageEntity $entity, string $type): string
```

Comment:
Naming is always hard. Seeing the function, o more logic name to me would be `composeContactString` as you return exactly that composed out of th data.